frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh

```diff
@@ -50,7 +54,10 @@ EOF
 
 # Install virtctl binary and add in to PATH
 mkdir virtctl
-wget https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/virtctl-${KUBEVIRT_VERSION}-linux-x86_64 -O virtctl/virtctl
+
+wget ${VIRTCTL_AMD64} -O virtctl/virtctl || wget ${VIRTCTL_X86_64} -O virtctl/virtctl
```

Comment:
what does this line do ?
you download the amd-64 binary and then if the download fails you download the x86 one ?

Reply:
@rawagner i may be missing something, can you take a look ^^